src/types.ts

```diff
@@ -0,0 +1,12 @@
+type MockDataItem = {
```

Comment:
Widzę że to dopiero zamockowane typy, ale lepiej mimo wszystko mieć specjalny folder `types` a w nim tworzyć pliki dedykowane dla każdego interfejsu/typu/enuma.